cclib/parser/molcasparser.py

```diff
@@ -656,7 +663,18 @@ def extract(self, inputfile, line):
                             if float(i) != 0:
                                 homos += 1
 
-                    line = next(inputfile)
+                    aonames = []
+                    if line.split():
+                        if line.split()[0] == '1':
+                            while line.split() != [] and line.split()[0] != '--':
+                                aonames.append("%s_%s" % (line.split()[1], line.split()[2]))
```

Comment:
While I think this is fine, you should favor being more explicit (i.e. self-documenting) when possible:

```
aonames.append("{atom}_{orbital}".format(atom=line.split()[1], orbital=line.split()[2]))
```